lib/cli/src/commands.rs

```diff
@@ -11,11 +11,15 @@ mod self_update;
 mod validate;
 #[cfg(feature = "wast")]
 mod wast;
+#[cfg(unix)]
```

Comment:
Binfmt can't be installed on macOS, so we should use the linux target directly.

```suggestion
#[cfg(target_os = "linux")]
```